### ISSUE TITLE
Performance degradation with Fenix A320

### ISSUE THREAD

**Issue by sushiat** [Sequal32/yourcontrols#157]
Hi, I recently installed yourcontrols and did a flight with a friend using the FBW and that worked pretty well and was great fun.

But when flying solo using the Fenix the performance of the sim degrades to a practically unflyable state (fps dipping below 10 and no higher then 15) from my normal 45. This happens around 1hr-1hr20mins into the flight and has happened on all of my last 3 flights.

I don't even have the windows application opened, so it's coming from the module in the community folder. Not sure if that is trying to connect the whole time in the background and is causing a memory leak or piling up exceptions, but it seems to get worse and worse as time goes on.

I haven't tested it with other aircraft and the shared flight we did was barely an hour so I didn't notice any degradation, or because we were actually using it, it didn't run into the same issue.

I will re-test it with the Fenix and the community module removed - hopefully I will be able to land this bird now and it will be next leg for testing, not redoing this one :)

**Comment by Midcon113** [comment]
This is most likely just the Fenix and nothing to do with YourControls. I
have this problem with virtually every Fenix flight.

**Comment by sushiat** [comment]
I really doubt that, in the month since getting the Fenix I have done well over 50 flights with it and I never encountered this issue before. Some of those flights were also over 4 hours long. I only had this issue on the last 3, and the only two things I changed were installing yourcontrols and updating my nvidia drivers. I reverted the drivers, did a clean install and it changed nothing, and then I remembered that I updated them after the first flight showing this problem. I will do another flight now this evening without yourcontrols to verify.

**Comment by Midcon113** [comment]
It’ll be interesting to hear your results. I’ve done the same flight with
the same route each time and had the same results with and without
yourcontrols being in the community folder.

**Comment by sushiat** [comment]
Just finished a return flight from EIDW-EGPH-EIDW, in total over 2:30 hours, performance back to normal

**Comment by Midcon113** [comment]
Excellent- thanks for sharing. I will try the same tonight and report.

**Comment by Midcon113** [comment]
Well I took out YourControls and sure enough you were correct. Going to do
some more flights to be sure but that appears to have made improvements! I
stand corrected - thanks for bringing this up!

**Comment by sushiat** [comment]
Glad it helped, got the same feedback from one of the guys in my VA - he was getting ready to chuck his PC out the window :)

Hopefully if/when this project comes out of hiatus the Fenix will be supported or at least no longer broken, so leaving this issue open here for reference.

**Comment by Midcon113** [comment]
Well, according to my testing, YC is clearly causing major issues. I hope this is fixed in the near future. In the meantime, moving YC out of the community folder appears to solve a great amount of performance degradations I’ve run into for a long time. Again, thank you for bringing this up! Sent from Mail for Windows From: Mark LamSent: Wednesday, July 6, 2022 16:14To: Sequal32/yourcontrolsCc: Comment; Sequal32/yourcontrolsSubject: Re: [Sequal32/yourcontrols] Performance degradation with Fenix A320 (Issue #157) Excellent idea! On Wed, Jul 6, 2022 at 13:42 sushi.at ***@***.***> wrote:Glad it helped, got the same feedback from one of the guys in my VA - he was getting ready to chuck his PC out the window :)Hopefully if/when this project comes out of hiatus the Fenix will be supported or at least no longer broken, so leaving this issue open here for reference.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

**Comment by LogRol** [comment]
Just a small note from me: this does happen with the A32NX as well. I've tried to fly around 2,5 hours with YourControls installed but not using it, and I ended up closing the flight before descent as the FPS was so low. Not sure if this happens even when activly using YourControls, did anyone experience the same FPS issues when using YC?

**Comment by vbr74** [comment]
Hello, I can confirm that, I've tried to fly with the Fenix A320 several dozens of time during 2 weeks when I've finally been able to figure out what the problem was : YourControls plug ins still activated in my addo-ons folder.  For information it was surely an outaded version, so maybe a fix has been issued for the root cause since.

I've disabled it and I've been able to fly a short haul domestic flight without any issues, and with ultra graphics and full add-ons (vPilot,REX Weather force, Self Loading Cargo, Navigraph, smartCARS for VA PIREP). 

Btw, I'm very intersted to add the Fenix in future update ith this case can been fixed. 

And also, I can try to reproduce the error and submit log to help the dev out there, and I've got some dev skills so If I can help to bring Fenix A320 compatibility I'm your man !
